Cambio-TontoDelDia

JLeonN opened the pull request.

JLeonN committed:
Preparando la estructura

JLeonN committed:
Estructura - 2

JLeonN committed:
Estructura - 3

francosang commented on frontend/src/Pagina/Inicio.jsx:
```diff
@@ -4,10 +4,10 @@ import Tarjeta from "../Componentes/Tarjeta/Tarjeta";
 
 const Inicio = () => {
-  const { cargaa } = useNombres();
+  const { carga } = useNombres();
```
Hay que borrar estas dos líneas porque te dan conflictos

JLeonN committed:
Actualizando componete a CowboyDelDia

También se creó un HOOK para el componente y se agregó la URL a la parte de la API. 🛠️

JLeonN committed:
ubicación

Pequeño error de ubicación. 📍😊

francosang commented on frontend/src/Pagina/Inicio.jsx:
```diff
@@ -1,16 +1,9 @@
 
 const Inicio = () => {
-  const { cargaa } = useNombres();
   const { tonto } = useObtenerTonto();
```
Hay algo raro acá. No aparecen mis cambios.

Fijate el PR que hice yo: https://github.com/The-Cowboys/the-cowboy-day/pull/51

Por qué no aparecen mis cambios acá???

francosang commented on frontend/src/Pagina/Inicio.jsx:
```diff
@@ -1,16 +1,9 @@
-  const { cargaa } = useNombres();
   const { tonto } = useObtenerTonto();
 
-  if (cargaa) {
```
Vos mismo agregaste este componente de carga, ya no lo querés?

Por qué lo estás borrando?

No me descanse 😂

francosang replied:
Capaz que no entendes completamente cómo funciona esto.

Cada hook tiene su variable de carga y sus datos.
Los hooks son independientes entre ellos y pueden estar los dos cargando a la vez.

Si vos querés mostrar un componente de carga para un hook tenes que usar la variable de carga de ese hook.

En este caso estabas usando la variable de carga de un hook y los datos de otro hook diferente.

francosang commented:
Hace dos cosas:
1. Actualiza tu rama con main para traerte los últimos cambios
2. Volvé a poner el componente de carga

JLeonN committed:
Merge branch 'main' of https://github.com/The-Cowboys/the-cowboy-day into Cambios-TontoDelDia

JLeonN commented:
Creo que ya quedó franco.

JLeonN committed:
Detalles borrar el componente

Me olvidé de borrar el componente que no se usa. 🤦‍♂️💻

francosang reviewed (approved):
Sabeee 👏 👏
